castai/resource_node_configuration.go

```diff
@@ -233,6 +240,15 @@ func resourceNodeConfiguration() *schema.Resource {
 							Description:      "Number of IPs per prefix to be used for calculating max pods.",
 							ValidateDiagFunc: validation.ToDiagFunc(validation.IntBetween(0, 256)),
 						},
+						FieldNodeConfigurationEKSImageFamily: {
```

Comment:
Could you please adjust `TestAccResourceNodeConfiguration_eks` to also use `eks_image_family`?